packages/react/src/components/footer/footerBase/FooterBase.tsx

```diff
@@ -34,7 +94,38 @@ export const FooterBase = ({ children, copyrightHolder, copyrightText }: FooterB
           )}
         </div>
       )}
-      {children && <div className={styles.links}>{children}</div>}
+      <div className={styles.baseActions}>
+        {children && (
+          <div className={styles.links}>
+            {Children.map(children, (child, index) => {
```

Comment:
Children.map will be deprecated in future react versions so it would be good to replace it with some other iteration method.

Reply:
Well that's a bit inconvenient as this pattern is used throughout my implementation 😄 Although it has felt a bit hacky and apparently isn't a very recommended pattern. I'll put a link here as a reminder on how to do this better (TLDR; hooks): https://www.reddit.com/r/reactjs/comments/10m4dr8/comment/j64kkii/

Reply:
Good to know 👍 
It seems that the whole Children - class/module will be deprecated: https://beta.reactjs.org/reference/react/Children
I think it harms a bit of the React usage. One alternative is to use a separate prop:
Instead of:
```jsx
 <Nav>
   <Link>
   <Link>
 </Nav>   
```
We should use:
```jsx
  <Nav links=[<Link />, <Link />] />
```

Reply:
Great Scott! You're right, I just might use this because I dumped the context idea because it's a bit cumbersome and not clear to developers and would not work particularly well in SSR.

Reply:
Also, yes the whole Children API will be legacy and there is quite a lot of usage of it within HDS (like the whole getChildren.ts file that many other components use). There will have to be a big "remppa".

Reply:
Check it out, I refactored away from Children in different ways. In this case I removed the Children.map and added `separator` boolean prop to NavigationLink to display the separating line. This means the separator could be added where it's not designed to be (navigation, utilities) but that's on the users in my opinion. I wanted to keep the unified practice of putting FooterNavigationLinks inside the Footer sections so I didn't go with `Footer.Base links={[<Footer.NavigationLink>...`.